.gitignore

```diff
@@ -45,3 +45,4 @@ test-support/sbt_cache/
 **/.ammonite/
 **/metals.sbt
 **/pom.xml
+/gh-release-token.txt
```

Comment:
Maybe a more general **/gh-release-token.txt to prevent us from including this file from ANY directory.

Reply:
done